styles/style.css

```diff
@@ -0,0 +1,526 @@
+    font-style: italic;
+}
+
+#personal, #professional {
+    z-index: 1;
+    margin-top: 50px;
+    padding: 20px;
+    box-shadow: -10px -10px white, 10px 10px var(--pink);
+}
+
+#professional a:hover, #projects-header a:hover {
+    color: var(--pink);
+}
+
+#personal ul {
+    padding: 0;
+    list-style: none;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+
+#personal li {
+    background-color: var(--dark-blue);
+    color: var(--mint);
+    height: fit-content;
+    width: fit-content;
+    padding: 15px;
+    border-radius: 20px;
+    margin-bottom: 10px;
+    margin-right: 10px;
+}
+
+/*************************/
+/* PORTFOLIO PAGE STYLES */
+/*************************/
+.pair {
+    font-size: smaller;
+    font-weight: bold;
+}
+
+.pair::before {
+    content: "‣";
+    padding-right: 10px;
+}
+
+.pair a {
+    background-color: var(--mint);
+    color: var(--dark-blue);
+    padding: 0 5px;
+    margin-left: 3px;
+}
+
+.pair a:hover {
+    background-color: var(--pink)
+}
+
+.pair a::after {
+    display: none;
+}
+
+time {
+    text-transform: uppercase;
+    font-weight: bold;
+    font-size: smaller;
+}
+
+.yay {
+    font-weight: 500;
+}
+
+#hi {
+    text-align: center;
+}
+
+#hey {
+    font-weight: 600;
+}
+
+#projects h2 {
+    background-color: black;
+    color: var(--mint);
+    border-bottom: 1px solid white;
+    text-align: center;
+    transition-property: background-color;
+    transition-duration: .08s;
+}
+
+#projects h2 + * {
+    margin-top: 30px;
+}
+
+#projects h2:hover {
+    background-color: var(--pink);
+}
+
+#projects h2 a {
+    display: block;
+    text-decoration: none;
+    width: 100%;
+    height: 100%;
+    padding: 10px;
+}
+
+#projects article {
+    background-color: var(--light-blue);
+}
+
+#projects article :is(div, p) {
+    padding-left: 40px;
+    padding-right: 40px;
+}
+
+#projects article:is(#adagrams, #viewing-party) {
+    margin-bottom: 0;
+}
+
+.description {
+    padding-bottom: 40px;
+}
+
+#projects h2 {
+    padding: 0;
+}
+
+#projects article a:hover {
+    border-color: inherit;
+}
+
+#projects article:is(:first-child, :last-child) {
+    background-color: rgb(40, 48, 68);
+    color: white;
+}
+
+code {
+    background-color: var(--mint);
+    border-radius: 5px;
+    padding: 3px;
+    display: inline-block;
+    color: var(--pink);
+}
+
+/*****************/
+/* MEDIA QUERIES */
+/*****************/
+
+/* DESKTOP SIZES */
+@media screen and (min-width: 1225px) {
+    #redwoods {
+        position: relative;
+    }
+    
+    #redwoods img {
+        position: absolute;
+        z-index: -1;
+        left: 10%;
+        width: 35%;
+    }
+
+    #personal {
+        margin-left: auto;
+        width: 50%;
+        background-color: var(--mint);
+    }
+    
+    #professional {
+        margin-left: auto;
+        margin-right: auto;
+        background-color: var(--mint);
+        width: 65%;
+    }
+}
+
+@media screen and (min-width: 1030px) {    
```

Comment:
Cool use of media queries! It's less important than it used to be, but consider using "only screen" to better support older browsers. See discussion [here](https://stackoverflow.com/questions/8549529/what-is-the-difference-between-screen-and-only-screen-in-media-queries)